conf/routes

```diff
@@ -30,6 +30,7 @@ GET     /logout                     controllers.AuthController.logout
 GET     /passkey/registration-options   controllers.PasskeyController.registrationOptions
 GET     /passkey/auth-options           controllers.PasskeyController.authenticationOptions
 POST    /passkey/register               controllers.PasskeyController.register
+POST    /passkey/delete                 controllers.PasskeyController.deletePasskey
```

Comment:
Would be better as a `DELETE`:
```suggestion
DELETE    /passkey                 controllers.PasskeyController.deletePasskey
```

Reply:
Only POST and GET methods are supported by HTML forms: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/form#method, so I'm going to leave it as is for now.

Reply:
Perhaps should add a comment to the route so that we know why this is a post